.github/workflows/krew.yaml

```diff
@@ -1,9 +1,8 @@
-name: prereleaser
+name: releaser
 on:
-  push:
-    tags:
-      - 'v*'
-      - '!v*-rc*'
+  release:
+    types: 
+      - published
```

Comment:
Will this be triggered on rc release? Krew does not publish the release candidate.